src/Day.jsx

```diff
@@ -182,7 +181,7 @@ export default class Day extends PureComponent {
       } else {
         // stretch element
         const startPos = positionInDay(date, selection.start, timeZone);
-        const minPos = startPos + (minLengthInMinutes * MINUTE_IN_PIXELS);
+        const minPos = startPos + minLengthInMinutes * MINUTE_IN_PIXELS;
```

Comment:
This change is causing a build failure, eslint is complaining about mixing `+` and `*`. If you run `npm run lint` locally you should see the same failure.